src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -619,9 +619,16 @@ sub check_dither {
     my $self = shift;
 
     my $dthr = shift;		  # Ref to array of hashes containing dither states
+    my $backstop = shift;
     my %dthr_cmd = (ON => 'ENAB',   # Translation from OR terminology to dither state term.
 		    OFF => 'DISA');
 
+    my %standard_y_dither = (20 => 1087.0,
```

Comment:
Need to include the other dither cases as well, probably encoded in a table like the one Erik provided.  That's my plan for hopper anyway.
```
Dither Case        Y Amplitude  Z Amplitude  Y Period  Z Period
-----------------  -----------  -----------  --------  --------
Standard ACIS           8.0          8.0      1000.0     707.1
Standard HRC           20.0         20.0      1087.0     768.6
Crab Observations       1.0          1.0      1000.0     707.1
ACIS Subarray           8.0          4.0      1000.0     707.4
ACIS Contaminant*       8.0         64.0      1000.0    2647.0
ACA DCC DFC             0.0          0.0      1000.0     707.1
```